unittests/generic/test_interval_tree.cpp

```diff
@@ -53,33 +53,34 @@ struct test
 
 INTERVAL_TREE_DECLARE_C(test, struct test, node, foo, bar);
 
-void printf_node(void *_n)
+void printf_node(void *n_)
 {
-	struct test *n = (struct test *)_n;
+	struct test *n = (struct test *)n_;
 	struct avl_node *p;
-	struct interval_tree_node *_p;
-	struct test *__p;
+	struct interval_tree_node *p_;
+	struct test *p__;
```

Comment:
I suggest to [rename also this identifier](https://stackoverflow.com/questions/228783/what-are-the-rules-about-using-an-underscore-in-a-c-identifier#answer-228797 "What are the rules about using an underscore in a C++ identifier?").